drivers/vcluster/vcluster.go

```diff
@@ -76,9 +126,19 @@ func DeleteVCluster(vclusterName string) error {
 	return nil
 }
 
+// TerminateVCluster Terminates Vcluster and runs it in it's own context
+func (v *VCluster) TerminateVCluster() error {
+	cmd := exec.Command("vcluster", "delete", v.Name)
```

Comment:
Can we have a common method for executing vCluster commands ?

Reply:
Done. Thanks for suggesting this optimization.